script.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+mkdir -p /var/lib/node_exporter 
+chown -Rv 200:200 /var/lib/node_exporter
+
+cat  > /etc/systemd/system/smart.service << EOF
+
+[Unit]
+Description=SMART
+
+[Service]
+Type=simple
+ExecStart=/bin/sh -c "/usr/bin/smartmon.sh -r > /var/lib/node_exporter/smartmon.prom"
+SyslogIdentifier=Smartmon
+Restart=always
+RestartSec=300
```

Comment:
Let's change this to run once per day (86400)..

Reply:
done